exercises/practice/book-store/src/test/java/BookStoreTest.java

```diff
@@ -119,6 +132,7 @@ public void twoCopiesOfEachBook() {
 
     @Disabled("Remove to run test")
     @Test
+    @DisplayName("Three copies of first book and two each of remaining")
     public void threeCopiesOfFirstBookAnd2EachOfRemaining() {
```

Comment:
```suggestion
    public void threeCopiesOfFirstBookAndTwoEachOfRemaining() {
```

Can you change the `2` in the method name to `Two`, just to be consistent with the rest of the method names!